drodrpg/DROD/GameScreen.cpp

```diff
@@ -8643,7 +8652,10 @@ void CGameScreen::DisplayAdjacentTempRoom(const UINT direction)
 		default: ASSERT(!"Unsupported temp room pan direction"); return;
 	}
 
-	if (!this->pCurrentGame->IsRoomAtCoordsExplored(newRoomX, newRoomY)) {
+	const UINT dwNewRoomID = this->pCurrentGame->pLevel->GetRoomIDAtCoords(newRoomX, newRoomY);
+
+	if (!this->pCurrentGame->IsRoomAtCoordsExplored(newRoomX, newRoomY) &&
```

Comment:
Now that you have dwNewRoomID, you can simplify this check to
CDbSavedGame::IsRoomExplored(dwNewRoomID);
which is the logic that IsRoomAtCoordsExplored includes, and it avoids the now-redundant call to GetRoomIDAtCoords.